suite/tests/CMakeLists.txt

```diff
@@ -3013,6 +3013,10 @@ else (UNIX)
   tobuild(security-win32.except-guard security-win32/except-guard.c)
   tobuild(security-win32.thread-setcontext security-win32/thread-setcontext.c)
   tobuild(security-win32.singlestep security-win32/singlestep.c)
+  if (NOT X64)
+    # no way to set debug register in 64 bit
+    tobuild(security-win32.except-debugreg security-win32/except-debugreg.c)
+  endif (NOT X64)
```

Comment:
Well, there is indeed a weakness here but I did not know how to fix it :
Most of the code works on 64-bit as well.
The only part that is 32-bit only is the test code setting the debug registers.
I did not find how to get this behavior on 64 bit.

So, what should be done ?